Source/WebKit/GPUProcess/graphics/RemoteGraphicsContextGL.cpp

```diff
@@ -211,7 +211,7 @@ void RemoteGraphicsContextGL::surfaceBufferToVideoFrame(WebCore::GraphicsContext
     assertIsCurrent(workQueue());
     std::optional<WebKit::RemoteVideoFrameProxy::Properties> result;
     if (auto videoFrame = protectedContext()->surfaceBufferToVideoFrame(buffer))
-        result = m_videoFrameObjectHeap->add(videoFrame.releaseNonNull());
+        result = Ref { m_videoFrameObjectHeap }->add(videoFrame.releaseNonNull());
```

Comment:
Ditto for adding `protectedVideoFrameObjectHeap()`

Reply:
Same as above.